frontend/test/pytest/test_template.py

```diff
@@ -34,7 +57,11 @@ def amplitude_embedding(f: jax.core.ShapedArray([4], float)):
     params = jax.numpy.array([1 / 2] * 4)
     interpreted_fn = qml.QNode(amplitude_embedding, device)
     jitted_fn = qjit(interpreted_fn)
-    assert np.allclose(interpreted_fn(params), jitted_fn(params))
+
+    interpreted_result = interpreted_fn(params)
```

Comment:
Is there a reason why you are putting `interpreted_fn(params)` and `jitted_fn(params)` into their own variables and then comparing them? Is the current version `np.allclose(interpreted_fn(params), jitted_fn(params))` failing?

Reply:
Also curious about this

Reply:
I copy/pasted an existing test as a template for my tests, and it did this. I went through and removed the redundant code on all the test.